.github/workflows/pr-manage.yml

```diff
@@ -0,0 +1,35 @@
+name: PR Manage Action
+
+on:
+  check_suite:
+  pull_request:
+    types:
+      - labeled
+      - opened
+      - ready_for_review
+      - reopened
+      - synchronize
+      - unlabeled
+      - unlocked
```

Comment:
This is a lot of triggers.... And this workflow does not look like it needs this many...